raven-android/src/main/java/com/getsentry/raven/android/event/helper/AndroidEventBuilderHelper.java

```diff
@@ -119,10 +123,36 @@ public void helpBuildingEvent(EventBuilder eventBuilder) {
         osMap.put("kernel_version", KERNEL_VERSION);
         osMap.put("rooted",         isRooted());
 
+        // App
+        PackageInfo packageInfo = getPackageInfo(ctx);
+        if (packageInfo != null) {
+            appMap.put("app_version", packageInfo.versionName);
+            appMap.put("app_build", packageInfo.versionCode);
+            appMap.put("app_identifier", packageInfo.packageName);
+        }
+
+        appMap.put("app_name", getApplicationName(ctx));
```

Comment:
Yeah, it's what we do with the other keys (and what iOS does). I guess the idea is that having a `null` key at least tells us we tried to set it but something bombed out (since there's no logging on mobile it's a helpful hint)?